tests/smoke/test_launch_states.py

```diff
@@ -142,7 +142,7 @@ def test_wrong_api_base_url(args):
     MagicMock(return_value=""),
 )
 def test_wrong_node_end_point(args):
-    args.node_endpoint = "https://testnet-tezos.giganode.io:4433"
+    args.node_endpoint = "https://api.tzkt.io/v1:4433"
```

Comment:
the URL should be taken from the constants list. I think it is the wrong port by intention in this test.

Reply:
yeah I'm certain this is the wrong port by intention because of the name of the function.